package.json

```diff
@@ -72,7 +72,7 @@
     "webpack-merge": "^4.2.2"
   },
   "dependencies": {
-    "@centreon/ui": "github:centreon/centreon-ui#adbb81c3b4dc6bf67842b890c2ac4e175562decd",
+    "@centreon/ui": "github:centreon/centreon-ui#MON-5016-style",
```

Comment:
We need to be careful to merge the corresponding centreon-ui branch before merging this one to avoid breaking the build afterwards